Jenkinsfile.cloud

```diff
@@ -83,27 +72,33 @@ node(NODE) {
     stage("Prepare Configs") {
         withCredentials([
             string(credentialsId: params.OSTREE_INSTALL_URL, variable: 'OSTREE_INSTALL_URL'),
-            string(credentialsId: params.INSTALLER_TREE_URL, variable: 'INSTALLER_TREE_URL'),
         ]) {
-            sh "sed -i 's,\\(<url>\\).*\\(<\\/url\\),\\1${INSTALLER_TREE_URL}\\2,' rhcos.tdl"
             sh "sed -i 's,@@OSTREE_INSTALL_URL@@,${OSTREE_INSTALL_URL},' cloud.ks"
             sh "sed -i 's,@@OSTREE_INSTALL_REF@@,${ref},' cloud.ks"
-            sh "rm -rf /var/lib/imagefactory/storage/*"
         }
     }
 
-    stage("Running imagefactory") {
-        sh """
-            imagefactory --debug base_image \
-                --file-parameter install_script cloud.ks \
-                --parameter offline_icicle True \
-                rhcos.tdl
+    // Note this runs unprivileged to avoid the virt stack trying to use privileges
+    // We run in bwrap which sets up a pid namespace +init to ensure qemu is reaped,
+    // which libvirt relies on.
+    image = "/tmp/rhcos.qcow2"
```

Comment:
Does this need a `def image` before `stage("Prepare Configs")`?  It looks like it is used in the `Postprocessing` stage, too

Reply:
Oh whoops...I see this is outside of the `Prepare Configs` stage.  Still  needs to be `def`'ed here, though?